ckan/logic/action/get.py

```diff
@@ -2897,7 +2897,7 @@ def display_name(followee: dict[str, Any]) -> Optional[str]:
         '''Return a display name for the given user, group or dataset dict.'''
         display_name = followee.get('display_name')
         fullname = followee.get('fullname')
-        title = followee.get('title')
+        title = plugins.toolkit.h.get_translated(followee, 'title')
```

Comment:
I've never used this endpoint. Does a change this simple really work here?

Reply:
I will test it out some more to make sure that it works for groups and orgs, and this does not break following users.

Reply:
@wardi okay this does not work, but for the same reason that the Dashboard group and organization route do not work.

These things just do some queries and puts the results directly into the dictization methods, and returns that. So these things do not go through any `plugin_validate` stuff, for any further processing??

Reply:
As discussed, we will grab the relative `group_show`, `organization_show`, or `package_show` in the followee stuffs.